python-squid/src/main/java/org/sonar/python/tree/PyTree.java

```diff
@@ -39,8 +40,8 @@ protected PyTree() {
   }
 
   @Override
-  public boolean is(Kind kind) {
-    return kind == getKind();
+  public boolean is(Kind... kinds) {
+    return Arrays.asList(kinds).contains(getKind());
```

Comment:
this will be heavily called, can we just make a simple loop ? 

```
Kind treeKind = kind();
    for (Kind kindIter : kinds) {
      if (treeKind == kindIter) {
        return true;
      }
    }
    return false;```